libs/vkd3d/state.c

```diff
@@ -2657,7 +2657,7 @@ STATIC_ASSERT(sizeof(struct vkd3d_shader_transform_feedback_element) == sizeof(D
 
 static HRESULT d3d12_graphics_pipeline_state_create_render_pass_for_plane_mask(
         struct d3d12_graphics_pipeline_state *graphics, struct d3d12_device *device,
-        const struct vkd3d_format *dynamic_dsv_format,
+        uint32_t rtv_active_mask, const struct vkd3d_format *dynamic_dsv_format,
```

Comment:
we might wan to refactor this to use a struct at some point, rather than adding more arguments

Reply:
ye, it's getting a bit crowded. Save that for later.